server/api/roster/roster-entity.ts

```diff
@@ -135,6 +135,9 @@ export abstract class RosterEntity extends BaseEntity {
       value = getOptionalValue(column.displayName, row, 'string');
     }
 
+    // Convert empty strings to null.
+    value = value || null;
```

Comment:
Is there a built-in way for the CSV parser to do this for us? Most of them I've used have this as an option

Reply:
Looks like there is a way to do that via the csv parser. It might still be good to keep this conversion in place though in case we import from other file types without that parser option.